go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -112,8 +116,13 @@ func NewDoltChunkStore(ctx context.Context, nbf *types.NomsBinFormat, org, repoN
 		},
 	})
 	if err != nil {
+		evt.SetAttribute(eventsapi.AttributeID_REMOTEAPI_ERROR, err.Error())
```

Comment:
I don't know that you'll want to capture the error in an attribute. Might be better to just have a counter to count errors per rpc.

Reply:
Same thing for each error case below.